utils/src/main/kotlin/DeclaredLicenseProcessor.kt

```diff
@@ -112,5 +112,5 @@ data class ProcessedDeclaredLicense(
      * The list of all mapped and unmapped licenses.
      */
     @JsonIgnore
-    val allLicenses = spdxExpression?.licenses().orEmpty() + unmapped
+    val allLicenses = spdxExpression?.decompose().orEmpty().map { it.toString() } + unmapped
```

Comment:
Commit message nit: You use different ticks in ´cyclone dx`, but how about anyway:

"Previously, the Static HTML and Cyclone DX reporters completely ignored ..."

Reply:
Done.